### DISCUSSION TITLE
Why do I report an error when pulling live streams? Do you all know the reason?

### DISCUSSION THREAD

**Discussion by kylin-li** [ant-media/Ant-Media-Server#6118] [Q&A]
Why do I report an error when pulling live streams? Do you all know the reason?
![Uploading 20240222095222.png…]()

**Comment by kylin-li** [comment]
avcodec error: Failed to open rtmp://192.168.2.113/MMH01/f1GOPvAaaVmA2NDC1702731613152: Operation not permitted
avcodec error: Failed to open rtmp://192.168.2.113/MMH01/f1GOPvAaaVmA2NDC1702731613152: Operation not permitted
avcodec error: Failed to open rtmp://192.168.2.113/MMH01/f1GOPvAaaVmA2NDC1702731613152: Operation not permitted

**Comment by kylin-li** [comment]
![Uploading 20240222095222.png…]()

**Comment by USAMAWIZARD** [comment]
@kylin-li can you please share me the output of this command 

ffprobe rtmp://192.168.2.113/MMH01/f1GOPvAaaVmA2NDC1702731613152

also try to play the stream with ffplay and see if it works

ffpllay rtmp://192.168.2.113/MMH01/f1GOPvAaaVmA2NDC1702731613152
